var/spack/repos/builtin/packages/py-currentscape/package.py

```diff
@@ -14,7 +14,7 @@ class PyCurrentscape(PythonPackage):
     git = "ssh://bbpcode.epfl.ch/cells/currentscape"
 
     version("develop", branch="master")
-    version("0.0.2", tag="0.0.2")
+    version("0.0.6", tag="currentscape-v0.0.6")
```

Comment:
Just a note : if user is not going to install old versions then it’s fine to remove old versions. Otherwise we do keep few important ones in the recipe.

Reply:
Thank you, I'll keep that in mind. I don't think any user would want to install version 0.0.2, it was very buggy.